doc/error-handling.md

```diff
@@ -1,11 +1,11 @@
-# Error Handling
+### Error Handling
 
 By default, if no strategy is defined, when an exception happens the process switches to the `failed` stage remembering
-the stage it was trying to execute and exits.
+the stage it was trying to execute and exists.
```

Comment:
This seems to revert a previous edit: the process `exits` after recording the failed stage.